src/main/java/ai/tecton/client/request/GetFeaturesRequestData.java

```diff
@@ -155,10 +162,11 @@ boolean isEmptyRequestContextMap() {
     return this.requestContextMap.isEmpty();
   }
 
-  private void validateKeyValue(String key, Object value) {
+  private void validateKeyValue(String key, Object value, boolean allowNullValue) {
```

Comment:
Super nitty but I'd like to see this as two separate methods `validateKeyValueAllowingNullValue` and `validateKeyValueDisallowingNullValue` cause I have trouble reading boolean params to methods